Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -1098,6 +1150,11 @@ protected class ClassWriter : IClassWriter {
           staticCallString = $"{DafnyRuntimeModule}.euclidian_modulus"; break;
 
         case BinaryExpr.ResolvedOpcode.EqCommon:
+          opString = "=="; break;
```

Comment:
What's the point?

Reply:
For all the non collection types `EqCommon` is used.

Reply:
Sure, but you didn’t group it in when I made that comment.

Reply:
Yes, it didn't cross my mind to check for it first.